src/main/java/com/openshift/internal/restclient/capability/resources/AbstractOpenShiftBinaryCapability.java

```diff
@@ -152,9 +154,11 @@ public final void start(final OpenShiftBinaryOption... options) {
 	}
 	
 	private void startProcess(final String location, final OpenShiftBinaryOption... options) {
-		String cmdLine = new StringBuilder(location).append(' ').append(buildArgs(Arrays.asList(options))).toString();
-		String[] args = StringUtils.split(cmdLine, " ");
+		List<String> args = new ArrayList<String>();
+		args.add("oc");
```

Comment:
I think you mean `location`, not `cmdLine`. Yes, i think it's a good proposal. Will change it. Though anyway we hard code such things, for example: https://github.com/openshift/openshift-restclient-java/blob/83f04c2e9c5a7dedef205b5c19faed46de098737/src/main/java/com/openshift/internal/restclient/capability/resources/OpenShiftBinaryRSync.java#L144